### DISCUSSION TITLE
What is the collar and overlap setting for computing diarization benchmarks

### DISCUSSION THREAD

**Discussion by ivangtorre** [pyannote/pyannote-audio#1798] [Development]
For reproducibility,
What are the parameters of collar and skip_overlap used to compute the diarization benchmark table?
Are those values obtained by estimating the number of speakers?

pyannote.metrics.diarization.DiarizationErrorRate(collar=¿?, skip_overlap=¿?)

**Comment by hbredin** [comment]
collar=0.0

skip_overlap=False
